docs/internal-developers/testing/releases/911.md

```diff
@@ -0,0 +1,262 @@
+7. Edit the Product Catalog and disable the `Inherit Query from template` option.
+8. Ensure all the settings that customize the query are visible.
+
+#### Post/Page Editor
+
+1. Add the Product Query in a template (eg: Product Catalog)
+2. Check that the `Inherit Query from template` option isn't visible.
+
+### Fix the incorrect layout of Rating and Price in Classic Template and Products block ([7932](https://github.com/woocommerce/woocommerce-blocks/pull/7932))
+
+Prerequisites:
+
+- make sure you have at least one product _with_ rating
+
+Steps:
+
+1. Go to Editor and add blocks:
+
+- Filter by Rating
+- All Products,
+- Products
+- Classic Template
+
+2. In the frontend check the above blocks
+
+**Expected:**
+
+- The layout of product in Products and Classic Template is as on the screenshots above
+- The layout of the Filter by Rating and All Products doesn't change
+
+### Product Elements: Fix block settings ([7914](https://github.com/woocommerce/woocommerce-blocks/pull/7914))
+
+1. Create a page and add the **Products (Beta)** block.
+2. Add **Button**, **Price** and **Rating** product elements.
+3. Customize via block settings:
+
+- **Product Price**: `background color` and `appearance`.
+- **Product Button**: `font-weight` and `padding`.
+- **Product Rating**: `font-size`, `margin`, `color`.
+
+4. Ensure all customizations display correctly both in the editor and the page (frontend).
+
+### StyleAttributesUtils: Fix PHP notices and invalid CSS output ([7909](https://github.com/woocommerce/woocommerce-blocks/pull/7909))
+
+1. Create a page and add the **Products (Beta)** block.
+2. Add the **Product Image** and **On-Sale Badge** blocks.
+3. Try setting different combinations of the following block settings:
+
+- Border Radius
+- Border Width (test also in conjunction with different Border Radius settings )
+- Margin
+- Padding
+
+Make sure to test in both 'Linked' and 'Unlinked' modes ( one value for all 4 sides, custom values for each side, some sides not set etc. ).
+
+Make sure for any of the scenarios no PHP errors and notices are being generated and the options display correct visual results on the page (frontend).
+
+### Update MiniCart.php to fix price total amount on page load prices are including tax ([7832](https://github.com/woocommerce/woocommerce-blocks/pull/7832))
+
+1. Go to WooCommerce > Settings and set _Display prices during cart and checkout_ to _Including tax_.
+![Settings screenshot](https://user-images.githubusercontent.com/3616980/206694476-39bda20b-c653-4703-978d-3d7f746c2bf5.png)
+2. Create a post or page and add the Mini Cart block.
+3. In the frontend, hover the Mini Cart block with the mouse and notice the price doesn't update ([before](https://user-images.githubusercontent.com/3616980/206694129-c16fcea4-8ac4-4bd7-a72a-221946c3ef08.webm) and [after](https://user-images.githubusercontent.com/3616980/206694060-eee1cccf-1ebd-435d-995d-23caca715918.webm)).
```

Comment:
My fault for not specifying this, but I think we can improve these testing steps:
```suggestion
1. In WooCommerce > Settings check _Enable tax rates and calculations_.
2. In the _Tax_ tab that appears go to _Tax options_ and  set _Display prices during cart and checkout_ to _Including tax_.
3. In _Standard rates_, create a tax rate with `Rate %` of `10`.
4. Create a post or page and add the Mini Cart block.
5. In the frontend, hover the Mini Cart block with the mouse and notice the price doesn't update ([before](https://user-images.githubusercontent.com/3616980/206694129-c16fcea4-8ac4-4bd7-a72a-221946c3ef08.webm) and [after](https://user-images.githubusercontent.com/3616980/206694060-eee1cccf-1ebd-435d-995d-23caca715918.webm)).
```